content/en/docs/concepts/workloads/pods/downward-api.md

```diff
@@ -75,8 +77,7 @@ via either mechanism are:
 `status.podIP`
 : the pod's primary IP address (usually, its IPv4 address)
 
-In addition, the following information is available through
-a `downwardAPI` volume `fieldRef`, but **not as environment variables**:
+The following information is available through a `downwardAPI` volume `fieldRef`, **but not as environment variables**:
```

Comment:
```suggestion
The following information is available through a `downwardAPI` volume `fieldRef`,
**but not as environment variables**:
```

Reply:
I have addressed the changes as you suggested. PTAL!!